pytest_inmanta/plugin.py

```diff
@@ -1130,3 +1142,16 @@ def finalize_handler(self, handler: ResourceHandler) -> None:
     def finalize_all_handlers(self) -> None:
         for handler_instance in self._handlers:
             self.finalize_handler(handler_instance)
+
+
+@pytest.fixture(scope="session")
+def session_temp_dir(tmpdir_factory: "TempdirFactory") -> Generator[str, None, None]:
```

Comment:
A shorter notation in case the  `SendType` and `ReturnType` are `None`:
```suggestion
def session_temp_dir(tmpdir_factory: "TempdirFactory") -> Iterator[str]:
```